pom.xml

```diff
@@ -258,6 +258,12 @@
             <version>5.6.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Nice, I've skipped the use of parametrised tests, but this seems like a good place to start using them.